packages/ds-component/src/types.ts

```diff
@@ -661,6 +668,7 @@ export interface TableFormat {
   fields: FieldsByConfigId;
   style: StyleById;
   tables: Tables;
+  dateRanges: DateRange;
```

Comment:
based on my suggestion above this should change to dateRanges: `DateRange | undefined`